storage/storage.go

```diff
@@ -37,17 +37,35 @@ type Retries struct {
 
 // Heartbeat is used for the ruby sidekiq web ui
 type Heartbeat struct {
-	Identity string
+	Identity string `json:"identity"`
 
-	Beat  time.Time
-	Quiet bool
-	Busy  int
-	RttUS int
-	RSS   int64
-	Info  string
-	Pid   int
+	Beat            int64  `json:"beat,string"`
+	Quiet           bool   `json:"quiet,string"`
+	Busy            int    `json:"busy,string"`
+	RttUS           int    `json:"rtt_us,string"`
+	RSS             int64  `json:"rss,string"`
+	Info            string `json:"info"`
```

Comment:
For symmetry, do you mind adding the `string` modifier to all of them?

Reply:
I attempted doing this, but cannot apply to string type:
https://go.dev/play/p/D7Phyjkwm3f

Reply:
Ohhhh right! Silly marshaller